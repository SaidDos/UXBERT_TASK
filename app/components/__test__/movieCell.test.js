import 'react-native';
import React from 'react';
import MovieCell from '../cells/movieCell';

import renderer from 'react-test-renderer';

it('should renders correctly', () => {
  const movie = {id: 1, title: 'said', year: '2030'};
  const tree = renderer.create(<MovieCell movie={movie} />).toJSON();
  expect(tree).toMatchSnapshot();
});
