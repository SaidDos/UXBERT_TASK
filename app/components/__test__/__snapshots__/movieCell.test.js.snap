// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "lightgrey",
      "flexDirection": "row",
      "height": 100,
      "marginBottom": 20,
      "padding": 10,
      "width": 319.3,
    }
  }
>
  <Image
    source={
      Object {
        "uri": undefined,
      }
    }
    style={
      Object {
        "alignSelf": "center",
        "height": 80,
        "justifyContent": "flex-start",
        "width": 80,
      }
    }
  />
  <View
    style={
      Object {
        "marginHorizontal": 10,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 16,
          "fontWeight": "bold",
        }
      }
    >
      said
    </Text>
    <Text
      style={
        Object {
          "color": "grey",
        }
      }
    >
       
      Released Date: 
    </Text>
    <Text
      style={
        Object {
          "color": "grey",
        }
      }
    >
       Year: 
      2030
    </Text>
  </View>
  <View
    style={
      Object {
        "bottom": 5,
        "flexDirection": "row",
        "position": "absolute",
        "right": 10,
      }
    }
  >
    <Text
      allowFontScaling={false}
      onPress={[Function]}
      style={
        Array [
          Object {
            "color": "#800080",
            "fontSize": 30,
          },
          Object {
            "paddingHorizontal": 10,
          },
          Object {
            "fontFamily": "Ionicons",
            "fontStyle": "normal",
            "fontWeight": "normal",
          },
          Object {},
        ]
      }
    >
      Ôçπ
    </Text>
  </View>
</View>
`;
